### PR TITLE
Broadcast 

### PR DESCRIPTION
**Pull request by archit-2003**

### CODE REVIEW

**Review by Ankur-Agrawal-ece20 — CHANGES_REQUESTED** [review]

File: customauth/views.py [hunk 1]
```diff
@@ -1,94 +1,183 @@
...
+
+class UserInitApi(generics.GenericAPIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    class InputSerializer(serializers.Serializer):
+        email = serializers.EmailField()
+        name = serializers.CharField(required=True)
+        college_name = serializers.CharField(required=True)
+        year = serializers.CharField(required=True)
+        phone_number = serializers.CharField(required=True)
+
```
[the remark this answers is not in the document]
Comment (Samar1110) [reply customauth/views.py:137]: added serializer

**Review by Samar1110 — COMMENTED** [review]
Done All changes

File: customauth/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ankur-Agrawal-ece20) [reply customauth/views.py:137]: It should be outside InputSerializer

**Review by Ankur-Agrawal-ece20 — CHANGES_REQUESTED** [review]
Update inputserializer part

**Pull request merged**